odin.R

```diff
@@ -0,0 +1,345 @@
+
```

Comment:
It looks like this is related to, but generated independently from, odin.Rmd ? we might automate that?  Can you comment about how much manual change there is, and if you think that would be feasable (as in, how "automatic" did it feel when you did the translation).

If they are in fact quite different can you describe where please?

Reply:
It was largely removing the markdown bits (and some of the text) around the R bits. The bits where we compile models with `odin()` were copied directly from the slides rather than the qmd, so I'm not entirely sure how automatically it could be done

Reply:
Have a go with `knitr::purl` and see if that does the job.  Otherwise we can probably do it with with `quarto inspect` - it's not super important though but it will be annoying to keep it in sync